src/libsync/connectionvalidator.cpp

```diff
@@ -235,6 +237,13 @@ void ConnectionValidator::slotCapabilitiesRecieved(const QVariantMap &json)
     auto caps = json.value("ocs").toMap().value("data").toMap().value("capabilities");
     qDebug() << "Server capabilities" << caps;
     _account->setCapabilities(caps.toMap());
+
+    QString serverVersion = caps.toMap()["core"].toMap()["status"].toMap()["version"].toString();
+    if (!serverVersion.isEmpty()) {
+        _account->setServerVersion(serverVersion);
+        qDebug() << "Version from capabilities instead of status.php:" << serverVersion;
```

Comment:
The thing is that adding debug output only make it looks more like that matrix screen, even for me.   While people who care to see the server version will be able to parse it from other debug output already